test/reference.html

```diff
@@ -907,14 +990,28 @@
                 }
                 j += 2;
             }
-        } else {
+        } else if (sections[i + 2] === 'wide') {
+            while (j < entries.length) {
+                if (entries[j] === sections[i]) {
+                    // entry matches the current section
+                    t += '<div class="wide-entry">';
+                    t += '<div><code>' + entries[j + 1] + '</code></div>';
+                    t += '<div class="mathquill-static-math">' + entries[j + 1] + '</div>';
+                    t += '<div class="codecogs"><img src="' + 'http://latex.codecogs.com/svg.latex?' + encodeURI(entries[j + 1]) + '"></div>';
+                    t += '<div class="mathjax">$$' + entries[j + 1] + '$$</div>';
+                    t += '</div>';
+                }
+                j += 2;
+            }
+       } else {
             t += '<table>';
             while (j < entries.length) {
                 if (entries[j] === sections[i]) {
                     // entry matches the current section
                     t += '<tr>';
                     t += '<td class="mathquill-static-math">' + entries[j + 1] + '</td>';
-                    t += '<td><img src="' + 'http://latex.codecogs.com/svg.latex?' + encodeURI(entries[j + 1]) + '"></td>';
+                    t += '<td class="codecogs"><img src="' + 'http://latex.codecogs.com/svg.latex?' + encodeURI(entries[j + 1]) + '"></td>';
+                    t += '<td class="mathjax">$$' + entries[j + 1] + '$$</td>';
```

Comment:
does this mean we're relying on the MathJax feature where it searches through every text node on the page for `$$`? Since we're generating the elements it might be a lot faster to call the MathJax API directly on our known elements (like by querying for all elements with the `.mathjax` class and MathJax-ifying them or something)